starter-code/iteration2/src/app/comments-component/comments-component.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-comments-component',
+  templateUrl: './comments-component.component.html',
+  styleUrls: ['./comments-component.component.css'],
+})
+export class CommentsComponentComponent implements OnInit {
+
+  title = 'Comments';
+  firstCommentName: string = 'Francesca';
+  firstComment: string = 'I went to Toronto this Summer. Absolutely lovely. Highly recommend it!';
+  secondCommentName: string = 'Michael';
+  secondComment: string = 'Meh. Imagine New York City, but cleaner.';
+
```

Comment:
So here we see some kind of structure and also we can see a repetition - there are some comments and they have the author (commentName) and some content. So better solution is to make something like you did in previous iteration (comments: Array<Object>;) and that will give us more flexibility and it's proper way to do it.